pyengine/katana.py

```diff
@@ -0,0 +1,58 @@
+def matcher(qty,book):
+
+    slices = []
+    slicefix = []
+
+    print(book)
+    print(qty)
+
+    while qty > 0:
+
+        # if there's no quotes, we return whatever we got beforehand
+        if len(book.keys()) == 0:
+            print('no quotes left')
+            return slices
+
+        # all of these variables should be replaced by the best possible option
+        lowestprice = 100000 # todo - first quote we look at should be the lowest price as opposed to this arbitrary high number
```

Comment:
obviously not going to work if the lowest price is greater than 100000, probably worth just fixing now